pkg/gui/filetree/commit_file_tree_view_model.go

```diff
@@ -80,7 +80,14 @@ func (self *CommitFileTreeViewModel) GetSelectedItemId() string {
 }
 
 func (self *CommitFileTreeViewModel) GetSelectedItems() ([]*CommitFileNode, int, int) {
-	panic("Not implemented")
+	startIdx, endIdx := self.GetSelectionRange()
```

Comment:
Can you add a check at the top of this function which returns `nil, 0, 0` if Len() is 0, like I do in this commit: https://github.com/jesseduffield/lazygit/commit/75d471df04d2333a8f809fd241beb2efa2a54138